src/database/member.rs

```diff
@@ -27,4 +27,8 @@ impl Member {
             },
         }
     }
+
+    pub fn equals(&self, id: &str) -> bool {
+        self.id.to_string() == id
+    }
```

Comment:
PartialEq トレイト及び Eq トレイトを実装すると、直接 == を使って比較できるようになります

Reply:
Member と &str の比較なので無理なのでは……？

Reply:
```
impl PartialEq<&str> for Member {
    fn eq(&self, other: &str) -> bool {
        self.id == *other
    }
}
```
のようにできます

Reply:
あーーなるほど，と思ったんですけど等価じゃないのであんまやりたくないな

Reply:
```suggestion
    pub fn equals(&self, id: &str) -> bool {
        self.id  == id
    }
```
了解しました　ではこの修正だけ置いておきます。。。

Reply:
![image](https://user-images.githubusercontent.com/12772118/93292526-b8b91280-f820-11ea-80f1-33d2997986cc.png)

Reply:
self.idがStringだと思ってました　ころしてください